pkg/geo/geos/geos.cc

```diff
@@ -119,6 +120,7 @@ struct CR_GEOS {
   CR_GEOS_Area_r GEOSArea_r;
   CR_GEOS_Length_r GEOSLength_r;
   CR_GEOS_Centroid_r GEOSGetCentroid_r;
+  CR_GEOS_Interpolate_r GEOSInterpolateNormalized_r;
```

Comment:
can you explain why we're using `InterpolateNormalized`? if this is required, please rename `Interpolate_r` to `InterpolateNormalized_r`.

Reply:
Changed `Geos` function which we're using for interpolation. Previously it was  `GEOSInterpolateNormalized` now it is `GEOSInterpolate` and Updated their names according to `GEOSInterpolate`.

Initially, I was using `GEOSInterpolateNormalized` as it was directly satisfying the use-case as it works with fraction directly.
But later realised that It is wrapper function over `GEOSInterpolate` inside which we are computing `LineString's` length each time it called.
Which could be inefficient for `MULTIPOINT` case as It can be done just computing length only once.  
And there may be the case in future where we may require to locate a point at a certain distance. For which `GEOSInterpolate` will be the best option rather somehow using `GEOSInterpolateNormalized`